### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by yevpy**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    try:
+        cmd, source_file, second_file = command.split(maxsplit=3)
+    except ValueError:
+        raise
```
Comment (EdAlekseiev) [inline app/main.py:5]: Don't raise an exception here. You can print a message here and finish the function execution.

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
the command "cp file.txt new_file.txt" doesn't work
https://github.com/mate-academy/py-copy-file#:~:text=copy_file(%22cp%20file.txt%20new_file.txt%22)

File: app/main.py [hunk 2]
```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cmd, source_file, second_file = command.split(" ")
```
Comment (EdAlekseiev) [inline app/main.py:2]: `" "` is the default separator.
https://www.w3schools.com/python/ref_string_split.asp

Comment (EdAlekseiev) [inline app/main.py:2]:
```suggestion
    cmd, source_file, second_file = command.split()
```

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

**Review by YukihiroSM — APPROVED** [review]

**Pull request closed without merging**